randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2757,21 +2322,31 @@ Extra - asset_id: collision_camera_031
               Gravity Suit
               Flash Shift or Simple IBJ or Use Spin Boost
           After Ghavoran - Pulse Radar Enky and Ballspark
```

Comment:
The gravity requirements can be removed, with the addition of Gravity onto the connection from the lower tunnel to burenia teleport. The ballspark requirement should also be moved to Life Recharge as this right now makes it possible to come down here, get speed in spin boost room, and then go to the dock to burenia teleport without the ability to charge a speed.